lib/rules/max-line-length/README.md

```diff
@@ -108,3 +108,19 @@ a { color: pink; }
  */
 a { color: pink; }
 ```
+
+### `ignorePattern: ["/regex/"]`
+
+Ignore any line that matches the given regex pattern, regardless of whether it is comment or not.
+
+Given:
+
+```js
+["/^\\@import\\s+/"]
```

Comment:
I was not aware of t@ but it appears you're right. 

I changed it in the tests as well.